features/ultils/helper.ts

```diff
@@ -45,6 +49,7 @@ const replacer = (substring: string) => {
 
 export const createCsvFile = async (
   inputCsvObject: string[][],
+  workingDir?: string,
```

Comment:
In the `createCsvFile`, how about naming `baseDir` than `workingDir` so that we can know how that option will be used in the function?

Reply:
Oh, you're right.
It should be `baseDir`, not `workingDir`.
Thank you.